README.md

```diff
@@ -1,3 +1,14 @@
-# web3-ui
+# Welcome to Web3-UI (The UI library for Web3)
+
+Web3-UI has been developed keeping in mind the specific needs of web3 websites.
+
+## Installation
+
+Web3-UI is available as an [npm package](https://www.npmjs.com/package/web3-ui)
+```npm install web3-ui``` or ```npm i web3-ui```
```

Comment:
We seem to use `yarn`, so I'd suggest changing this to `yarn install`.

I'd also include information about running storybook:

"To start storybook and see the components already created:"
```
yarn storybook
```

Reply:
Thank you so much for the feedback @crondinini. Your points are totally valid, I will update the README with ```yarn storybook``` instruction and other necessary information so that this PR gets merged and closed. 🙌